BuildResidentialHPXML/measure.rb

```diff
@@ -1855,6 +1864,18 @@ def arguments(model)
     arg.setDescription('Year modules manufactured of the PV system 1.')
```

Comment:
Remove the `pv_system_year_modules_manufactured` arguments. It is used to calculate the system losses fraction, which you have an explicit input for. No need to ask for both.